core-services/image-mirroring/openshift/mapping_origin_4_5

```diff
@@ -8,6 +8,7 @@ registry.svc.ci.openshift.org/origin/4.5:aws-machine-controllers quay.io/openshi
 registry.svc.ci.openshift.org/origin/4.5:azure-machine-controllers quay.io/openshift/origin-azure-machine-controllers:4.5 quay.io/openshift/origin-azure-machine-controllers:4.5.0 quay.io/openshift/origin-azure-machine-controllers:latest
 registry.svc.ci.openshift.org/origin/4.5:baremetal-runtimecfg quay.io/openshift/origin-baremetal-runtimecfg:4.5 quay.io/openshift/origin-baremetal-runtimecfg:4.5.0 quay.io/openshift/origin-baremetal-runtimecfg:latest
 registry.svc.ci.openshift.org/origin/4.5:base quay.io/openshift/origin-base:4.5 quay.io/openshift/origin-base:4.5.0 quay.io/openshift/origin-base:latest
+registry.svc.ci.openshift.org/origin/4.5:bond-cni quay.io/openshift/origin-network-bond-cni:4.5 quay.io/openshift/origin-bond-cni:4.5.0 quay.io/openshift/origin-bond-cni:latest
```

Comment:
The naming on this is far too generic.  Your image name in the stream must be much more specific.  What is the function of this CNI driver, and is it part of a larger logical component?

Reply:
The image is used by the network operator (init container in the operator daemonset) to copy the bond-cni binary to the nodes.
The bond-cni only needs the binary, no additional configuration/daemonsets/... are needed.